src/components/layer-controller/LayerControllerSubscriber.js

```diff
@@ -83,6 +94,8 @@ function publishLayer({ loader, imageData, layerId }) {
 
 function LayerControllerSubscriber({ onReady, removeGridComponent, theme }) {
   const [imageOptions, setImageOptions] = useState(null);
+  const [cellsEvent, setCellsEvent] = useState(null);
+  const [moleculesEvent, setMoleculesEvent] = useState(null);
```

Comment:
A little confusing... `cellsEvent` seems like something that's transient, not a state? Perhaps it should just be `cells`?

Reply:
It's really just a flag for whether or not the event was received.  I think `cells` is good.

Reply:
@ilan-gold : Can you give it a boolean-ish name then? Thanks!

Reply:
Yes.